src/main/java/frc/robot/RobotContainer.java

```diff
@@ -19,45 +22,35 @@
  * subsystems, commands, and trigger mappings) should be declared here.
  */
 public class RobotContainer {
-  // The robot's subsystems and commands are defined here...
-  private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
 
-  // Replace with CommandPS4Controller or CommandJoystick if needed
-  private final CommandXboxController m_driverController =
-      new CommandXboxController(OperatorConstants.kDriverControllerPort);
+  /* Setting up bindings for necessary control of the swerve drive platform */
+  private final CommandPS4Controller joystick = new CommandPS4Controller(3);
+  private final SwerveSubsystem drivetrain = Constants.DriveTrain;
+  private final SwerveJoystickCommand swerveJoystickCommand = new SwerveJoystickCommand(
+        () -> -joystick.getRawAxis(1),
+        () -> -joystick.getRawAxis(0),
+        () -> -joystick.getRawAxis(2),
+        () -> (joystick.getRawAxis(4) + 1d) / 2d,
+        drivetrain
+        );
```

Comment:
Leave comments for what the Axis are on the actual ps4 controller 
ex: `// axis 4 corresponds to the R2 button`

Reply:
done